vita3k/interface.cpp

```diff
@@ -465,28 +465,33 @@ static ExitCode load_app_impl(SceUID &main_module_id, EmuEnvState &emuenv) {
             process_preload_disabled = *preload_disabled_ptr.get(emuenv.mem);
         }
     }
+
     const auto module_app_path{ emuenv.pref_path / "ux0/app" / emuenv.io.app_path / "sce_module" };
-    const auto is_app = fs::exists(module_app_path) && !fs::is_empty(module_app_path);
+    std::map<std::string, bool> is_module_in_app = { { "libc", {} }, { "libfios2", {} } };
+    for (auto &[module_name, is_app] : is_module_in_app)
+        is_app = fs::exists(module_app_path / fmt::format("{}.suprx", module_name));
+
     std::vector<std::string> lib_load_list = {};
     // todo: check if module is imported
     auto add_preload_module = [&](uint32_t code, SceSysmoduleModuleId module_id, const std::string &name, bool load_from_app) {
         if ((process_preload_disabled & code) == 0) {
             if (is_lle_module(name, emuenv)) {
+                const auto module_name_file = fmt::format("{}.suprx", name);
                 if (load_from_app)
```

Comment:
```suggestion
                if (load_from_app && fs::exists(module_app_path / fmt::format("{}.suprx", module_name)))
```
Just replace everything else by this and the line below.

Reply:
Not sur understund ?  Here code you write is check 2 time same things 🤔

Reply:
Oki, done, normaly 🤔